### PR TITLE
Run phpcs against our custom ruleset

### PR DESCRIPTION
**Pull request by john-n-smith**
Scrutinizer appear to have updated their version of phpcs meaning our ruleset is now compatible.
Not a fan of hardcoding the commit hash, but that the only way I could see of including it.

### CODE REVIEW

File: .scrutinizer.yml [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
-#     php:
-#         custom_coding_standard:
-#             git_repository: 'https://github.com/graze/standards'
-#             git_version: 'ff7ea57e14cc222ff2c29f36a0bf40e707783bbb'
-#             ruleset_path: 'PHP/CodeSniffer/Graze/ruleset.xml'
-
 checks:
     php:
+        custom_coding_standard:
+        git_repository: 'https://github.com/graze/standards'
```
Comment (h-bragg) [inline .scrutinizer.yml]: looks like it should be more indented.

**Comment by john-n-smith** [comment]
@h-bragg, sorted

**Comment by h-bragg** [comment]
Looks like it has gotten a bit confused and needs rebasing.

**Comment by john-n-smith** [comment]
@h-bragg, good spot. Sorted again.

**Comment by h-bragg** [comment]
👍

**Comment by john-n-smith** [comment]
@h-bragg, I held off committing this. I like the idea of Scrutinizer's global conf. 

Would you be a proponent of removing .scrutinizer files from our projects? Give me a 👍  and I'll create a PR for `graze/skeleton`

**Comment by h-bragg** [comment]
I think we should remove the file, if a project then needs custom settings it can add its own `.scruitinzer` file without many issues (maybe add a comment about it in the `README.md` or a new file `SERVICE_SETUP.md` that details how to link a project with github/travis/scruitinzer etc).

**Review by h-bragg — APPROVED** [review]
:+1:

**Pull request merged**